packages/core/src/lib/collapse.svelte

```diff
@@ -47,7 +47,7 @@ const handleClick = () => {
     role="button"
     aria-label="Toggle Content"
     tabindex="0"
-    class="flex w-full cursor-pointer items-center justify-between border border-light bg-white px-4 py-2 text-default"
+    class="border-light text-default flex w-full cursor-pointer items-center justify-between border bg-white px-4 py-2"
```

Comment:
[not for this PR] Oof, something is messed up with out prettier / eslint tailwind plugins. The fact that `border-light` and `text-default` got shunted to the front mean those plugins aren't reading our tailwind config properly.

Something to deal with sooner rather than later I think

Reply:
I have been seeing a lot of churn around class ordering so that makes sense. I can try to look into it.